### DISCUSSION TITLE
Help issue with the bot. *Video included*

### DISCUSSION THREAD

**Discussion by tech-degenerate** [jef/streetmerchant#1517] [Help]
This just started happening idk but to do. I have tried everything I know. I have uninstalled and reinstalled it. etc. 

https://user-images.githubusercontent.com/74688808/103148600-8e37f900-472f-11eb-83f1-b431a7e87c7d.mp4

**Comment by gigi2006** [comment]
Look Here
https://github.com/jef/streetmerchant/issues/1516#issuecomment-751335280

**Reply by tech-degenerate** [reply]
Thanks
